fineract-provider/src/main/java/org/apache/fineract/cob/api/ConfigureBusinessStepResource.java

```diff
@@ -69,6 +71,20 @@ public class ConfigureBusinessStepResource {
     private final ConfigJobParameterService configJobParameterService;
     private final PortfolioCommandSourceWritePlatformService commandWritePlatformService;
 
+    @GET
+    @Path("/names")
```

Comment:
Definitely not the proper naming for the API (`/jobs/names`). Please figure out a business step related API naming here.

Reply:
Same as above, plus we are using Configured Job Names for the others stuffs